go-controller/pkg/ovn/controller/services/utils.go

```diff
@@ -361,10 +361,20 @@ func collectServiceVIPs(service *v1.Service) sets.String {
 	for _, svcPort := range service.Spec.Ports {
 		// Node Port
 		if svcPort.NodePort != 0 {
-			for _, isIPv6 := range []bool{false, true} {
+			// slice to hold versions of ip families to check based on
+			// false for ipv4, true for ipv6
+			isV6Families := make([]bool, 0, 2)
```

Comment:
No need to do this in every step of the loop, you can build the slice outside.
Also a wrapper function that spits out the slice would be more readable

Reply:
thanks, will do that